Apps/MineOS/docker-compose.yml

```diff
@@ -0,0 +1,162 @@
+name: mineos-node
+# Base file from: https://github.com/hexparrot/mineos-node/blob/master/docker-compose.yml
+
+services:
+  mineos:
+    image: hexparrot/mineos:latest
+    container_name: mineos_server
+    networks:
+      - mineos
+    restart: unless-stopped
+
+    # Adjust as needed; if this is too short, stopping the container
+    # may corrupt map data
+    stop_grace_period: 10m
+
+    volumes:
+      # Using a bind mapping is preferrable here due to how docker
+      # stores names volumes by default in /var/lib/docker/volumes
+      # with permissions that prevent normal users from accessing
+      # the folder structure.
+      - /var/games/mineos/minecraft:/var/games/minecraft
+
+    environment:
+      # Should the web interface use HTTPS or HTTP?
```

Comment:
It'd be nice to move these comments to the corresponding `description` field under `x-casaos` field below.

This way, user could see the descriptions from UI directly in future (instead of jumping into code)